rollup.config.js

```diff
@@ -10,6 +10,24 @@ import { dependencies } from './package-lock.json'
 import fs from 'fs'
 import path from 'path'
 
+import * as edge from 'edge.js'
+edge.registerViews(path.join(__dirname, './packages/shell-chrome/views'))
+const renderPanel = () => {
+    let panel = edge.render('master')
+    let dir = path.join(__dirname, './dist/chrome')
+
+    if (process.env.NODE_ENV === 'production') {
+        panel = panel.replace(/:data-testid="[^"]*"/g, '')
+    }
+
+    if (!fs.existsSync(dir)) {
+        fs.mkdirSync(dir, { recursive: true })
```

Comment:
👍

I'm thinking we should ditch the whole `dist/chrome` stuff in favour of just `dist` 🤷🏼‍♀️ different PR I guess

Reply:
Do we also need to nest the codebase in `packages/shell-chrome` or could we move that up to `app` or `src`?

Reply:
yeah, technically it could just be `packages/extension` right? Since we use the same code for Chrome/Firefox.

Keeping structured like a monorepo makes some sense, since we've got the extension + the simulator (which are cool to have as separate "packages")